<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWindow</class>
 <widget class="QMainWindow" name="MWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resolucion de Ecuaciones</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Metodo:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cboMethod">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>115</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Biseccion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Iteracion de punto fijo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Newton</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Secante</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Posicion Falsa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Muller</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ecuacion:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leEquation">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>259</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupParameters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>721</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Parametros</string>
    </property>
    <widget class="QLineEdit" name="leParam4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbParam3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Param3:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leParam3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbParam4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Param4:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leParam2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leParam5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbParam2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Param2:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leParam1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbParam1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Param1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbParam5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Param5:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCalculate">
     <property name="geometry">
      <rect>
       <x>524</x>
       <y>82</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Calcular</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="teSteps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>721</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbResult">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>450</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>450</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Resultado:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCalculate_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Limpiar Historial</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leEquation</tabstop>
  <tabstop>cboMethod</tabstop>
  <tabstop>leParam1</tabstop>
  <tabstop>leParam2</tabstop>
  <tabstop>leParam3</tabstop>
  <tabstop>leParam4</tabstop>
  <tabstop>leParam5</tabstop>
  <tabstop>pbCalculate</tabstop>
  <tabstop>teSteps</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
